app/models/order_template.rb

```diff
@@ -0,0 +1,10 @@
+class OrderTemplate < ApplicationRecord
+  acts_as_tenant(:tenant)
+  include Discard::Model
+  default_scope -> { kept }
+  belongs_to :prepostable, :polymorphic => true
```

Comment:
Naming is hard.  Think we need to come up with something more readable.  I'll never remember what this is referring to.